Source/Anchorage.swift

```diff
@@ -338,6 +344,28 @@ public struct LayoutExpression<T : LayoutAnchorType> {
 
 }
 
+// MARK: - LayoutPriority
+
+public enum LayoutPriority {
+
+    case required
+    case high
+    case low
+    case fittingSize
+    case custom(Alias.LayoutPriority)
+
+    var priority: Alias.LayoutPriority {
```

Comment:
I prefer a name like `value` to avoid things like the `priority.priority` confusion below